rhscl.yaml

```diff
@@ -147,42 +201,42 @@ containers:
     expose: [80, 443]
   httpd24-php55:
     collection: httpd24
-    install: httpd24 php55 php55-php
+    install: httpd24 php55 php55-php @devel_libs
     expose: [80, 443]
   httpd24-python27:
     collection: httpd24
-    install: httpd24 python27
+    install: httpd24 python27 @devel_libs
     expose: [80, 443]
   httpd24-python33:
     collection: httpd24
-    install: httpd24 python33 python33-mod_wsgi
+    install: httpd24 python33 python33-mod_wsgi @devel_libs
     expose: [80, 443]
   perl516:
-    install: perl516
+    install: perl516 @devel_libs
   httpd24-perl516:
     collection: httpd24
-    install: httpd24 perl516 perl516-mod_perl
+    install: httpd24 perl516 perl516-mod_perl @devel_libs
     expose: [80, 443]
   nginx14:
-    install: nginx14
+    install: nginx14 @devel_libs
```

Comment:
Is there a need to install devel_libs here? (they are useful when you're building apps)